src/Http/Controllers/RegisteredUserController.php

```diff
@@ -46,15 +48,15 @@ public function create(Request $request): RegisterViewResponse
      *
      * @param  \Illuminate\Http\Request  $request
      * @param  \Laravel\Fortify\Contracts\CreatesNewUsers  $creator
-     * @return \Laravel\Fortify\Contracts\RegisterResponse
+     * @return \Laravel\Fortify\Contracts\RegisterResponse | \Illuminate\Http\RedirectResponse
      */
     public function store(Request $request,
-                          CreatesNewUsers $creator): RegisterResponse
+                          CreatesNewUsers $creator)
```

Comment:
This is a breaking change to the method signature.

Reply:
I know, but when, try login and failed, weee have an `\Illuminate\Http\RedirectResponse` and `Laravel\Fortify\Contracts\RegisterResponse` extend `Illuminate\Contracts\Support\Responsable`, we don't have no return in the same hierarchy to actually use polymorphism.

Reply:
Anyone only expecting a `RegisterResponse` will see their apps fail if something else is returned. This should be sent to master.

Reply:
Then, login only with the same rules, and always return `RegisterResponse`?

```PHP
    public function store(Request $request,
                          CreatesNewUsers $creator): RegisterResponse
    {
        event(new Registered($user = $creator->create($request->all())));

        $this->loginPipeline($request)->then(function () {
        });

        return app(RegisterResponse::class);
```

Reply:
This looks good?